### ISSUE TITLE
taskNoTasks message shows multiple times after task creation

### ISSUE THREAD

**Issue by FredrikSchaeferVitagroup** [hzi-braunschweig/SORMAS-Project#7327]
### Bug Description

See screenshot. The message appears one additional time for each created task.

### Steps to Reproduce

1. Open a case
2. Add a task and save

### Expected Behavior

Created tasks should either show, or the message should consistently just appear once.

### Screenshots

![image](https://user-images.githubusercontent.com/67001822/142587549-be9728cb-d697-471a-b0ef-8a0523cbdbe6.png)

### System Details

* Device: Windows
* SORMAS version: 1.66.0-SNAPSHOT
* Android version/Browser: Chrome
* Server URL: local dev system
* User Role: National User

**Comment by alexcaruntu-vita** [comment]
It can be reproduced with a user with a combination of the following roles: ReST User, Admin, Sormas to Sormas Client, National User, External Visits User.

**Comment by roxanamlendea** [comment]
Verified ticket on test-de.sormas.netzlink.com using the latest version of Sormas 1.66.0-SNAPSHOT. The issue reported here is fixed, but there's something related to this that I'm not sure of. Maybe @MateStrysewske know if this is expected or not. I created a user (ReST User, Admin, Sormas to Sormas Client, National User, External Visits User) and this user can create tasks, but he cannot see them. The tasks added by this user can be seen/edited by other users. My question is if is expected that this user can create tasks, but cannot see them?
